app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/learning_ui/LearningTestPresentation.kt

```diff
@@ -25,7 +25,7 @@ import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
-class LearningTest {
+class LearningTestPresentation {
```

Comment:
LearningPresentationModeTest as a name would be a bit clearer imo. I don't know if put the word "Test" at the end of a test class name is a good practice but I think it's more readable. (Same idea for Representation)

Reply:
yes, my b. LearningTest was the original name and I wanted to split it up into 2, but wasnt careful with naming